sar/core/graphshard.py

```diff
@@ -52,11 +51,6 @@
 logger.setLevel(logging.DEBUG)
 
 def bytes2human(n):
-    # http://code.activestate.com/recipes/578019
```

Comment:
@Kacper-Pietkun we might want to leave this in for legal purposes, or we could switch to using a python package to do the bytes to human readable formatting, e.g., https://pypi.org/project/humanfriendly/

Reply:
I changed function to use humanfriendly package (I pushed the changes to #10, because in that PR I refactored code responsible for memory tracking)